deepset_cloud_sdk/workflows/sync_client/pipeline_client.py

```diff
@@ -0,0 +1,125 @@
+"""Sync pipeline client for importing pipelines and indexes to deepset AI Platform."""
+import asyncio
+
+import structlog
+
+from deepset_cloud_sdk._service.pipeline_service import PipelineProtocol
+from deepset_cloud_sdk.models import IndexConfig, PipelineConfig
+from deepset_cloud_sdk.workflows.async_client.async_pipeline_client import (
+    AsyncPipelineClient,
+)
+
+logger = structlog.get_logger(__name__)
+
+
+class PipelineClient(AsyncPipelineClient):
```

Comment:
lets do composition and instanciate the async client. s.t we use the init + can use the same method name ?